### PR TITLE
Fixed policy of ifstatement in cognitive complexity

### PR DESCRIPTION
**Pull request by Vitaly-Protasov**
Fixed https://github.com/yegor256/aibolit/issues/215 by @lukyanoffpashok

### CODE REVIEW

**Comment by acheshkov** [comment]
@0crat out

File: aibolit/metrics/cognitiveC/cognitive_c.py [hunk 1]
```diff
@@ -44,6 +44,21 @@ def traverse_childs(self, block: Any, nested_level: int) -> None:
         for each_child in block.children:
             self.get_complexity(each_child, nested_level)
 
+    def __if_stat(self, expr, nested_level: int) -> None:
```
Comment (acheshkov) [inline aibolit/metrics/cognitiveC/cognitive_c.py:47]: what is the type of `expr` argument?

File: aibolit/metrics/cognitiveC/cognitive_c.py [hunk 2]
```diff
@@ -44,6 +44,21 @@ def traverse_childs(self, block: Any, nested_level: int) -> None:
...
+
+        if expr.else_statement is not None:
+            if isinstance(expr.else_statement, IfStatement):
+                self.complexity -= nested_level
+                self.__if_stat(expr.else_statement, nested_level)
+            else:
+                self.complexity += 1
+                self.get_complexity(expr.else_statement, nested_level + 1)
+
     def increment_logical_operators(self, block: BinaryOperation, prev_operator: str) -> None:
```
Comment (acheshkov) [inline aibolit/metrics/cognitiveC/cognitive_c.py:62]: all internal function of the class must have prefix `_`

File: aibolit/metrics/cognitiveC/cognitive_c.py (hunk 1 above)
Comment (acheshkov) [reply aibolit/metrics/cognitiveC/cognitive_c.py:47]: change name prefix `__`  to `_`

Comment (Vitaly-Protasov) [reply aibolit/metrics/cognitiveC/cognitive_c.py]: fixed

File: aibolit/metrics/cognitiveC/cognitive_c.py (hunk 2 above)
Comment (Vitaly-Protasov) [reply aibolit/metrics/cognitiveC/cognitive_c.py]: done

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/233#issuecomment-617591734))

@acheshkov Job `gh:yegor256/aibolit#233` is not assigned, can't get performer

**Pull request merged**